libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx

```diff
@@ -0,0 +1,220 @@
+  const {
+    data,
+    tabInformation,
+    children,
+    errors,
+    withLogsNavigation,
+    pauseLogs,
+    setPauseLogs,
+    lineNumbers,
+    enabledNginx,
+    setEnabledNginx,
+    clusterBanner,
+    countNginx,
+    applicationStatus,
+    placeholderDescription = 'Logs not available',
+  } = props
+
+  const location = useLocation()
+  const refScrollSection = useRef<HTMLDivElement>(null)
+  const [updateTimeContextValue, setUpdateTimeContext] = useState(defaultUpdateTimeContext)
+
+  const { organizationId = '', projectId = '', environmentId = '', serviceId = '' } = useParams()
+
+  const scrollToError = () => {
+    const section = refScrollSection.current
+    if (!section) return
+
+    const row = section.querySelector('.row-error')
+    if (row) scrollParentToChild(section, row, 100)
+  }
+
+  const LinkNavigation = (name: string, link: string, status?: StateEnum) => {
+    const isActive = location.pathname.includes(link)
+    return (
+      <Link
+        data-testid="nav"
+        className={`flex items-center h-full px-6 text-sm font-medium text-text-100 transition-colors transition-timing duration-250 hover:bg-element-light-darker-500 rounded-t ${
+          isActive ? 'bg-element-light-darker-400' : ''
+        }`}
+        to={link}
+      >
+        {status && <StatusChip status={status} mustRenameStatus className="mr-2" />}
+        <span className="truncate">{name}</span>
+      </Link>
+    )
+  }
+
+  return (
+    <div
+      className={`overflow-hidden flex relative w-full p-1 ${
+        clusterBanner ? 'h-[calc(100vh-8rem)]' : 'h-[calc(100vh-4.25rem)]'
+      }`}
+    >
+      {withLogsNavigation && (
+        <div className="absolute z-20 overflow-y-auto left-1 flex items-center w-[calc(100%-8px)] h-11 border-b border-element-light-darker-200 bg-element-light-darker-700">
+          {LinkNavigation(
+            'Deployment logs',
+            ENVIRONMENT_LOGS_URL(organizationId, projectId, environmentId) + DEPLOYMENT_LOGS_URL(serviceId)
+          )}
+          {LinkNavigation(
+            'Live logs',
+            ENVIRONMENT_LOGS_URL(organizationId, projectId, environmentId) + SERVICE_LOGS_URL(serviceId),
+            applicationStatus
+          )}
+        </div>
+      )}
+      {!data || data.items?.length === 0 || data?.loadingStatus === 'not loaded' ? (
+        <div data-testid="loading-screen" className="mt-[100px] w-full flex justify-center text-center">
+          {data?.loadingStatus === 'not loaded' || !data ? (
+            <LoaderSpinner className="w-6 h-6" theme="dark" />
+          ) : (
+            <div className="flex flex-col items-center">
+              <img
+                className="w-40 pointer-events-none user-none"
+                src="/assets/images/event-placeholder-dark.svg"
+                alt="Event placeholder"
+              />
+              <p className="mt-5 text-text-100 font-medium text-center">{placeholderDescription}</p>
```

Comment:
If possible,  we should take into account these conditions:
    - if depl history empty (environment never deployed before): "This service has never been deployed and no thus logs are available."
    - else (service deployed at least once but not this time)  "This service is not being deployed right now and thus no deployment logs are available. You can access the application logs from the "Live Logs" tab". Not sure if we can add a button to open the tab